CONTRIBUTING.md

```diff
@@ -12,6 +12,46 @@ These notes are built using [Docusaurus](https://docusaurus.io/), a modern stati
 
 - [Visual Studio Code](https://code.visualstudio.com/).  This is not strictly necessary, but we encourage it since we use various extensions to make working on the site easier.
 
+## Tutorial for Converting Table-like Diagrams into Images
```

Comment:
Let's move this section down below install instructions, etc.  Probably it can come last.

Reply:
All done! 😎